README.md

```diff
@@ -86,6 +85,23 @@ To run the test suite with test weights (only works in branch `solutions`) while
 
 **NOTE**: all of these tasks tasks take the `currentProject` setting into account, see above.
 
+### Building docker container images for grading
+
+To build a docker container image to upload to Coursera to use for grading, be sure to have the following prerequisites installed:
+
+ - [`docker`](https://docker.com) - A popular container system for Linux.
+ - [`courseraprogramming`](https://github.com/coursera/courseraprogramming) - A toolkit to help instructional teams build grading containers
+
+To dockerize an sbt project, copy the sample docker file in `docker/Dockerfile.sample` to `Dockerfile` at the root of your project. Also copy `docker/grade.sh.template` to the root of your project. Then run: `docker build -t <your-project-name> .` to build and tag your container image (e.g. `docker build -t reactive .` to tag it as `reactive`).
```

Comment:
Comment for later (we don't need to address this yet): we can (and should) eventually automate this part - a dependent SBT `dockerize` task can do these steps.